main.go

```diff
@@ -186,7 +188,7 @@ func SendPeriodicNotification(ctx context.Context, j *Job) error {
 		durString,
 	)
 
-	return sendNotif(ctx, j, j.Status, subject, msg, "analysis_periodic_notification")
+	return sendNotif(ctx, j, j.Status, subject, msg, j.NotifyPeriodic, "analysis_periodic_notification")
```

Comment:
I haven't actually moved the NotifyPeriodic into the notif_statuses table, because it's static anyway and I didn't really want to write another database migration. The interval is still there, and it's beneficial for it to be since it's used for filtering the list of jobs that may need notification.